dpnp/dpnp_iface_linearalgebra.py

```diff
@@ -115,6 +99,7 @@ def dot(x1, x2, **kwargs):
 
         if not (dim1 >= 2 and dim2 == 1) and not (dim1 >= 2 and dim2 >= 2) and (x1.dtype == x2.dtype):
             result = dpnp_dot(x1, x2)
+            print("hello")
```

Comment:
Please, remove such kind of things before commit. This behavior is not provided by `numpy` API